app.py

```diff
@@ -582,6 +626,13 @@ def adventure_page(adventure_name, level):
                     desired_level = key
         level = desired_level
 
+    # If a level is specified and user is logged in, check if there's a stored program available for this level
```

Comment:
Could we remove this block of additions? I think this came from a merge, but we don't need this anymore.

Reply:
Fixed!